src/config.c

```diff
@@ -367,6 +370,20 @@ void read_FTLconf(void)
 	if(buffer != NULL && strcasecmp(buffer, "false") == 0)
 		config.block_esni = false;
 
+	// NICE
+	// Shall we change the nice of the current process?
+	// defaults to: -10 (can be disabled by setting value to -999)
```

Comment:
Is there a reason for the default to be `-10`? This is a question we will be asked many times.

Reply:
No and Yes.

Quoting `man sched`:
>       With the advent of the CFS scheduler in kernel 2.6.23, Linux adopted
>       an algorithm that causes relative differences in nice values to have
>       a much stronger effect.  **In the current implementation, each unit of
>       difference in the nice values of two processes results in a factor of
>       1.25 in the degree to which the scheduler favors the higher priority
>       process.**  This causes very low nice values (+19) to truly provide
>       little CPU to a process whenever there is any other higher priority
>       load on the system, and makes high nice values (-20) deliver most of
>       the CPU to applications that require it (e.g., some audio applica‐
>       tions).

This means that a niceness of `-10` will result in roughly 9x more cycles given to `pihole-FTL` compared to other processes also asking for 100% CPU usages.